### DISCUSSION TITLE
Portable Version any help appreciated.

### DISCUSSION THREAD

**Discussion by Vetrro** [cyberofficial/Synthalingua#106] [Q&A] [answered]
When I setup and run the portable version I get this error after it downloads a few segments of a Twitch stream

![Capture](https://github.com/cyberofficial/Synthalingua/assets/61427255/7d49f46e-1239-4e85-bf24-ef58d3af2a96)

After the error it keeps downloading more segments but there is not translations available in the sub title window or the browser source

I have tried troubleshooting step for [WinError 2] issue where you install ffmpeg and manually add to path but I still get the same thing. If anyone knows what's up it would be greatly appreciated. Thanks :)

**Comment by cyberofficial** [comment]
hmm interesting 🤔

I will check out whats up and get back to this post.

Thanks for the report.

**Comment by cyberofficial** [comment, marked as answer]
Hello, Could you download `Version 1.1-A3` from https://cyberofficial.itch.io/synthalingua

When you launch the gui make sure to regenerate the command if you saved it, and make sure the new line marked in yellow shows (will vary depending on your where store the folder"  

![explorer_bwxFngRwdX](https://github.com/cyberofficial/Synthalingua/assets/19499442/6a330439-bd46-423b-b0b4-d9317e8ea437)

Looking for your feedback/reply.

**Comment by Vetrro** [comment]
It seems to be working now with the new version, Thanks for the quick response and fix :D

**Comment by cyberofficial** [comment]
Thanks for the feedback :) 
Feel free to reach out again if you need help or questions :]

**Comment by jedinjapan** [comment]
![image](https://github.com/cyberofficial/Synthalingua/assets/24239829/ca42a655-8a87-4378-a49f-3782b3237338)
Experiencing same/similar issue using the Alpha3 release as well. The command block shows ffmpeg being loaded from the path call "C:\Synthalingua\ffmpeg_path.bat" so I hope this isn't the issue. Any advice appreciated.

**Reply by cyberofficial** [reply]
does the folder Synthalingua contain a folder ffmpeg?
